tox.ini

```diff
@@ -0,0 +1,70 @@
+[tox]
+envlist =
+    lint,
+    test
```

Comment:
We also will want to lint the client, so add another item to the envlist `lintclient`, then a section like
```
[testenv:lintclient]
skip_install = true
usedevelop = false
deps =
changedir = {toxinidir}/girder_tulane/web_client
whitelist_externals =
  npm
commands =
  npm install --no-package-lock
  npm run lint
```

Reply:
Do you find this is a better approach than simply, e.g., invoking `yarn lint` or `npm run lint` in CI directly?

Reply:
See 19118408453d7a5ccbceffdfd146f362c8c32418

Reply:
Yes.  I like to be able to run all tests in one command (i.e., `tox`).

Reply:
Can you add `lintclient ` to the `envlist`?

Reply:
Done via d02e204.